src/generators/Minecraft.res

```diff
@@ -240,3 +240,101 @@ module MinecraftCharacter = {
     },
   }
 }
+
+type direction = [#Right | #Left | #Top | #Bottom]
+
+/* let drawCuboidLegacy = (
```

Comment:
What are your thoughts with this one?

Reply:
I feel phasing out the legacy functions and definitions might be in order soon, since all of the old generators have been added, and after `drawCuboid()` is added it should be easy to rewrite all of the generators using legacy functions that way. So, I'm not too confident on duplicating code for legacy functions if porting is essentially finished.